app/helpers/moments_helper.rb

```diff
@@ -146,24 +166,29 @@ def data_for(item)
     end
   end
 
-  def total_moment(total_count)
-    unless total_count == 1
-      return t(
-        'stats.total_moments', count: total_count
-      )
-    end
-
-    t('stats.total_moment', count: total_count)
+  def moment_count
+    {
+      total: current_user.moments.all.count,
+      monthly: current_user.moments.where(
+        created_at: Time.current.beginning_of_month..Time.current
+      ).count
+    }
   end
 
-  def monthly_moment(monthly_count)
-    unless monthly_count == 1
-      return t(
-        'stats.monthly_moments', count: monthly_count
-      )
+  def total_moment
+    if moment_count[:total] == 1
+      t('stats.total_moment', count: moment_count[:total])
+    else
+      t('stats.total_moments', count: moment_count[:total])
     end
+  end
 
-    t('stats.monthly_moment', count: monthly_count)
+  def monthly_moment
+    if moment_count[:monthly] == 1
+      t('stats.monthly_moment', count: moment_count[:monthly])
+    else
+      t('stats.monthly_moments', count: moment_count[:monthly])
```

Comment:
You could do something like this:
```
moment_key = moment_count[:monthly] == 1 ? 'moment' : 'moments'
t("stats.monthly_#{moment_key}", count: moment_count[:monthly])
```

Reply:
This is so neat @julianguyen . I am definitely doing it 😄